exercises/C-comparison-operators/exercise.js

```diff
@@ -7,14 +7,14 @@
 
 let studentCount = 16;
 let mentorCount = 9;
-let moreStudentsThanMentors; // finish this statement
+let moreStudentsThanMentors = true; // finish this statement
```

Comment:
Use the two variables above this line to do an expression and set the value of `moreStudentsThanMentors`